src/schema/home/context.ts

```diff
@@ -180,9 +181,78 @@ const Context: GraphQLFieldConfig<
       HomePageModuleContextTrendingType,
     ],
   }),
+  deprecationReason: deprecate({ inVersion: 2, preferUsageOf: "v2_context" }),
   resolve: ({ key, params }, _options, context) => {
     return moduleContext[key](context, params)
   },
 }
 
-export default Context
+const HomePageArtworkOfRelatedToFollowedArtistModuleType = new GraphQLObjectType<
+  any,
+  ResolverContext
+>({
+  name: "HomePageArtworkOfRelatedToFollowedArtistModule",
+  fields: () => ({
+    artist: {
+      type: Artist.type,
+    },
+    basedOn: {
+      type: Artist.type,
+      resolve: ({ based_on }) => based_on,
+    },
+  }),
+})
+
+const HomePageArtworkOfFollowedArtistModuleType = new GraphQLObjectType<
```

Comment:
@zephraph Perhaps this should just be `Artist` and we can add to the description of the `context` field that `Artist` represents a followed artist?

Reply:
Yeah, I'm not sure. The name is both very descriptive and a bit confusing so I'm definitely up for changing it. We could name the field `followedArtist` and just return the artist type?

Reply:
I don't know if my suggested made sense in the context of your question. I'd be wary of using a description on `context` as being the sole differentiator just because the description might be overlooked.

Reply:
Fair enough, as well. I’ll leave this as-is then 👍